src/inspector_agent.cc

```diff
@@ -658,9 +686,21 @@ bool Agent::StartIoThread(bool wait_for_connect) {
   }
 
   v8::Isolate* isolate = parent_env_->isolate();
+  HandleScope handle_scope(isolate);
+
+  // Enable tracking of async stack traces
+  if (!enable_async_hook_function_.IsEmpty()) {
+    Local<Function> enable_fn = enable_async_hook_function_.Get(isolate);
```

Comment:
Uff, I am afraid this is a bit over my head. Would it be a problem to land my pull request without any tests for `ConnectJSBindingsSession`?